superset-embedded-sdk/README.md

```diff
@@ -139,6 +139,14 @@ with the secret set in configuration variable `GUEST_TOKEN_JWT_SECRET` (see conf
 }
 ```
 
+In this example, the configuration file includes the following setting:
+
+```python
+GUEST_TOKEN_JWT_AUDIENCE="superset"
+```
+
+If you're using a different audience value, be sure to update the GUEST_TOKEN_JWT_AUDIENCE variable accordingly.
+
```

Comment:
Do we want to talk about the default value? Checking `superset/config.py`, it's declared like this:
``` python
GUEST_TOKEN_JWT_AUDIENCE: Callable[[], str] | str | None = None
```

Then it seems in the `security_manager` we have:
``` python
@staticmethod
def _get_guest_token_jwt_audience() -> str:
    audience = current_app.config["GUEST_TOKEN_JWT_AUDIENCE"] or get_url_host()
    if callable(audience):
        audience = audience()
    return audience
```

So it seems the URL host would be the default value if not specified in the config?

Reply:
I can add it, but personally I think its a little confusing. `get_url_host()` returns the value of another configuration variable - `WEBDRIVER_BASEURL` which also has its own default value. For the sake of simplicity I'd lean towards a recommendation of making clear that setting `GUEST_TOKEN_JWT_AUDIENCE` is recommended for using the SDK.

Reply:
awesome -- than you!

Reply:
Docs site PRs are warmly welcomed, if you want to add anything there :D

Reply:
Thanks, I added a bit of clarity there too.